### PR TITLE
Altoverview2

### PR DESCRIPTION
**Pull request by Mingling94**
Even more abstraction and less repetitious code than #24. :+1: or :-1:?

### CODE REVIEW

**Comment by DavidVorick** [comment]
I think this one is the best. Do you want me to merge it and close #24? (Luke is in agreement)

**Comment by Mingling94** [comment]
@DavidVorick @lukechampine
The Overview's javascript remains the same for the most part with the last commit, though I made a few corrections. 

Peruse the tutorial, specifically the last section, and merge if satisfied ^_^

File: doc/plugin-tutorial.md [hunk 1]
```diff
@@ -136,11 +136,11 @@ unique id's to update each of them separately later in JS.
...
 The plugin directory should now be:
 
-```text
-Sia-UI/app/plugins/Overview/
-├── index.html
-└── assets/
-    └── button.png
+```diff
+ Sia-UI/app/plugins/Overview/
++├── index.html
```
Comment (DavidVorick) [inline doc/plugin-tutorial.md:141]: this is magic

File: doc/plugin-tutorial.md [hunk 2]
```diff
@@ -259,14 +258,14 @@ Quite a lot to take in without review, but that's how styling webpages goes.
...
-└── css/
-    └── overview.css
+```diff
+ Sia-UI/app/plugins/Overview/
+ ├── index.html
+ ├── assets/
+ │   ├── button.png
++│   └── roboto-condensed-min.css
++└── css/
++    └── overview.css
```
Comment (DavidVorick) [inline doc/plugin-tutorial.md:268]: dark magic

File: doc/plugin-tutorial.md [hunk 3]
```diff
@@ -410,52 +420,57 @@ function init() {
...
-
 // Update values per call
 IPC.on('/wallet/status', function(err, result) {
-	balance = formatSiacoin(result.Balance) || balance;
-	document.getElementById('balance').innerHTML = 'Balance: ' + balance;
+	if (err) {
+		console.error(err);
+	}
+	if (result) {
+		document.getElementById('balance').innerHTML = 'Balance: ' + balance;
```
Comment (DavidVorick) [inline doc/plugin-tutorial.md]: don't you use `updateField` now?

**Comment by DavidVorick** [comment]
looks good, tutorial still doesn't cover jsdoc

**Pull request merged**